content/en/docs/File Format/implementationstatus.md

```diff
@@ -13,94 +13,96 @@ implementations.
 The value in each box means:
 * ✅: supported
 * ❌: not supported
+* (R/W): partial reader/writer only support
 * (blank) no data
 
 Implementations:
 * `C++`: [parquet-cpp](https://github.com/apache/arrow/tree/main/cpp/src/parquet)
 * `Java`: [parquet-java](https://github.com/apache/parquet-java)
 * `Go`: [parquet-go](https://github.com/apache/arrow-go/tree/main/parquet)
 * `Rust`: [parquet-rs](https://github.com/apache/arrow-rs/blob/main/parquet/README.md)
+* `CUDA C++`: [cudf](https://github.com/rapidsai/cudf)
```

Comment:
Should this be `cuDF`? Or `CUDA C++` is a more official name of it?

Reply:
`cuDF` is the name of the implementing dataframes library and `CUDA C++` is the language being used for implementation.

Reply:
I would prefer `cuDF` here. I think the original intention was to include implementations governed by the Parquet community or the Apache Software Foundation. It would be better to use the library name to encourage other Parquet implementations to appear here. WDYT? @alamb

Reply:
I also recall that the idea here was to list library names (so this would be better as `cuDF`) not languages. 

It just so happens that we only had one example library for each language so there was (before this PR) a 1-1 correspondence. 

Does that make sense @mhaseeb123 ?

Reply:
Sounds good. I will update this